modules/docs/markdown/03-protocols/04-simple-rest-json/01-overview.md

```diff
@@ -86,9 +86,20 @@ See the section about [unions](../../04-codegen/02-unions.md) for a detailed des
 * This limit can be changed by setting the maxArity `smithy4s.http4s.SimpleRestJsonBuilder.withMaxArity(.)` to the desired value.
 * an example can be seen in the [client example](03-client.md)
 
-## Explicit Null Encoding
+## Field Filtering and Explicit Null Encoding
```

Comment:
Some documentation also ought to be added to the serialisation section, after this PR : https://github.com/disneystreaming/smithy4s/pull/1653

Reply:
@ghostbuster91 the other PR has been merged, please adapt

Reply:
done :+1: